charts/incubator/frigate/templates/common.yaml

```diff
@@ -1,7 +1,7 @@
 {{/* Make sure all variables are set properly */}}
-{{- include "tc.common.loader.init" . }}
+{{- include "tc.v1.common.loader.init" . }}
 
 {{- include "frigate.configmap" . -}}
```

Comment:
{{/* Render configmap for frigate */}}
{{- $config := include "frigate.configmap" . | fromYaml -}}
{{- if $config -}}
  {{- $_ := mustMergeOverwrite .Values $config -}}
{{- end -}}